templates/.vscode/settings.json

```diff
@@ -7,5 +7,11 @@
     "[c]": {
         "editor.quickSuggestions": true
     },
-    "C_Cpp.intelliSenseEngineFallback": "Disabled"
+    "C_Cpp.intelliSenseEngineFallback": "Disabled",
+    
+    "idf.adapterTargetName": "Esp32_S2",
```

Comment:
I'm not sure, does this automatically get changed depending on the selected target?

Reply:
it does